synapse/config/workers.py

```diff
@@ -64,6 +64,10 @@ class WriterLocations:
     Attributes:
         events: The instances that write to the event and backfill streams.
         typing: The instance that writes to the typing stream.
+        to_device: The instance that writes to the to_device stream.
+        account_data: The instance that writes to the account data streams.
+        receipts: The instance that writes to the receipts stream.
+        presence: The instance that writes to the presence stream.
```

Comment:
looks like each of these can be a list of instances?

Reply:
well, apparently it's a single-element list, but you should probably say that.

(why are these different to `typing` ?)

Reply:
> (why are these different to typing ?)

Err, good Q. I think there are two options here: 1) make the single item lists just be strings, or 2) make typing be a single item list. I'm tempted to go with the latter so everything is consistent, but the former may be a bit less confusing.

Reply:
making typing a single-item list sounds fine to me.